src/System.Management.Automation/commands/utility/FormatAndOutput/out-console/OutConsole.cs

```diff
@@ -79,9 +79,14 @@ protected override void BeginProcessing()
                 mrt.MergeUnclaimedPreviousErrorResults = true;
             }
 
-            _savedTranscribeOnly = Host.UI.TranscribeOnly;
+            if (!_transcriptStateChanged)
+            {
+                _savedTranscribeOnly = Host.UI.TranscribeOnly;
+            }
+
             if (Transcript)
             {
+                _transcriptStateChanged = true;
```

Comment:
I am not going to say that nested or intersected `Out-Default -Transcript` are useful scenario, but if you are not support it, then (taking in account that `OutDefaultCommand` is the only place where `TranscribeOnly` modified) the only value that `_savedTranscribeOnly` will have is `false`. So, why bother and define variable, which will be always `false`?